collections/_article/godot-developer-fund.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Introducing the new Godot Development Fund"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in  funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
+ 
+
+## Reducing platform fees, amplifying impact
+
+The Development Fund will make The Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.
+
+If you are currently supporting us [through Patreon](https://www.patreon.com/godotengine), we encourage you to switch to the [Godot Development Fund](https://fund.godotengine.org) to increase the effectiveness of your donations.
+
+## Giving back to the community
+
+The support of the community is what keeps making it possible to reach [new heights](https://godotengine.org/article/godot-4-0-sets-sail/). With you, we can achieve even more!
+
+Every donation, no matter the size, helps us immensely! With your help we can hire more developers, create high quality demos, pay for hosting services and infrastructure, and make the best possible Godot Engine. The more funds we have, the more we can give back to you and the entire gamedev community.
+
+## Future enhancements
+
+While this is a good, fully functioning starting point for the platform, we will keep improving the Developer Fund.
+
+In the future, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms. 
+
+## Freedom and independence
+
+By relying more on user-funded contributions, we reduce our dependence on large grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open-source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.
```

Comment:
```suggestion
By relying more on recurring user-funded contributions and company sponsorships, we reduce our dependence on large one-time grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.
```

Maybe helps address @clayjohn's concern? We do need to encourage for corporate sponsorships _a lot_ more, which are usually upheld for multiple years and help us plan ahead.

Reply:
yeah, I like this edit